spec/features/admin/translations_spec.rb

```diff
@@ -219,18 +219,18 @@
 
   context "localization settings" do
     given(:language) { Spree.t(:this_file_language, scope: 'i18n', locale: 'de') }
-    given(:french) { Spree.t(:this_file_language, scope: 'i18n', locale: 'fr') }
 
     background do
       create(:store)
-      SolidusI18n::Config.available_locales = [:en, :'pt-BR', :fr]
-      visit spree.admin_locale_path
+      SolidusI18n::Config.available_locales = [:en, :'pt-BR']
+      visit spree.edit_admin_general_settings_path
+      click_on "Locales"
     end
 
-    scenario "adds german to available locales" do
+    scenario "adds german to supported locales" do
       targetted_select2_search(language, from: '#s2id_available_locales_')
       click_on 'Update'
-      expect(SolidusI18n::Config.available_locales).to include(:de)
+      expect(SolidusGlobalize::Config.supported_locales).to include(:de)
```

Comment:
Same comment as above. Please use my fix https://github.com/tvdeyen/solidus_globalize/commit/3683e7668d5443a27518b5c519968ccc4f6f2ce9

You are changing the implementation here.